api/node/routes_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"net/http"
 	"net/http/httptest"
 	"testing"
+	"time"
```

Comment:
Could you arrange the imports? And also, remove errs nickname from errors package.

Reply:
done